### DISCUSSION TITLE
Feature Request: Support for the SWR Package in FloPy

### DISCUSSION THREAD

**Discussion by eacuellarq** [modflowpy/flopy#2567] [Q&A]
Currently, FloPy provides extensive support for many MODFLOW packages, but it seems the Surface-Water Routing (SWR) package from MODFLOW-NWT is not included. The SWR package is crucial for modeling surface-water and groundwater interactions, and having it integrated into FloPy would be highly beneficial for many users.

I would like to request the following:

Official Support for SWR: If feasible, adding SWR to FloPy would significantly enhance its capabilities.
Guidance for Custom Implementation: If official support is not planned, could you provide documentation or guidelines for extending FloPy to include unsupported packages like SWR?
Potential Benefits:

Expands FloPy’s functionality for coupled surface-water/groundwater modeling.
Allows for streamlined workflows when using MODFLOW-NWT with SWR.
Willing to Contribute:
I am happy to assist by testing or providing feedback if SWR support is developed.

Additional Context:
If there are known workarounds or plans for this in the future, any information would be greatly appreciated.

**Comment by wpbonelli** [comment]
There is a `ModflowSwr1` class in the `flopy.modflow.mfswr1` module, I see that [while we have a notebook about post-processing SWR](https://flopy.readthedocs.io/en/stable/Notebooks/load_swr_binary_data_example.html) results it doesn't show the package itself
